Gemfile

```
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.1'
```

- change all single quote to double quote
- remove all comment lines (start with `#` character)
ex:
```suggestion
ruby "3.0.1"

gem "rails", "~> 6.1.4"
gem "sqlite3", "~> 1.4"
etc.
```